src/components/CldImage/CldImage.vue

```diff
@@ -86,7 +94,7 @@ export default {
       this.imageLoaded = true;
     },
     renderImageOnly(src, hasPlaceholder = false) {
-      const imgClass = `${IMAGE_CLASSES.DEFAULT} ${!this.imageLoaded ? IMAGE_CLASSES.LOADING : IMAGE_CLASSES.LOADED}`
+      const imgClass = `${IMAGE_CLASSES.DEFAULT} ${!this.imageLoaded ? IMAGE_CLASSES.LOADING : IMAGE_CLASSES.LOADED} ${this.imgClass}`
```

Comment:
Please note that this adds an empty space at the end in case `this.imgClass` is empty, and thus our tests in Travis are failing (You can see the breaking tests on the pull request page).

Maybe we can trim extra spaces at the end to avoid this problem?

Reply:
I've added a ternary operation that add the class if it is not empty, it seems to works fine.

Reply:
Thanks, one last thing - can you please add a test that confirms that this feature works as expected? 
I'd imagine a test that uses this new propr and confirms that the class appears on the right element in the DOM to be enough.